Dockerfile

```diff
@@ -1,6 +1,48 @@
-ARG ROS2_VERSION=humble
-FROM ghcr.io/aica-technology/ros2-ws:${ROS2_VERSION} as base
-USER ${USER}
+ARG BASE_TAG=22.04
+FROM ubuntu:${BASE_TAG} as base
+ENV DEBIAN_FRONTEND=noninteractive
+ENV USER developer
+ENV HOME /home/${USER}
+
+RUN apt-get update && apt-get install -y \
+    cmake \
+    g++ \
+    git \
+    libgtest-dev \
+    libeigen3-dev \
+    python3-pip \
+    ssh \
+    sudo \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/*
+
+RUN echo "Set disable_coredump false" >> /etc/sudo.conf
+
+# create and configure a new user
+ARG UID=1000
```

Comment:
Do we even need a user? Is that only for debugging? In that case can it be in another stage so the main build path don't need it?

Reply:
I tried without, but yeah we need one for the devcontainer. But good idea, I will try to move it further down